CMakeLists.txt

```diff
@@ -223,17 +223,17 @@ add_subdirectory(docs)
 
 # DESTINATION gets the previous path and appends the argument --> /opt/local/GrPPI
 
-# cmake -DCMAKE_INSTALL_PREFIX=path/to/folder
+# cmake .. -DCMAKE_INSTALL_PREFIX=path/to/folder
```

Comment:
Is the syntax of how to change the default installation path. It may be useful for the user but we can remove it.